internal/third_party/kubernetes/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
I believe we don't need to include the complete license. The file headers mention the license, and the source where they came from, which should be sufficient.

Reply:
Done.